ggml/src/ggml-cuda/mmq.cu

```diff
@@ -110,7 +110,7 @@ void ggml_cuda_mul_mat_q(
     const int64_t s3  =  dst->nb[3] / ts_dst;
 
     const bool use_stream_k = ((GGML_CUDA_CC_IS_NVIDIA(cc) && ggml_cuda_highest_compiled_arch(cc) >= GGML_CUDA_CC_VOLTA)
-                            || (GGML_CUDA_CC_IS_AMD(cc) && GGML_CUDA_CC_IS_CDNA3(cc)));
+                            || GGML_CUDA_CC_IS_CDNA(cc));
```

Comment:
Not that this was introduced with this PR, but the outmost parentheses have no effect here. Please move the closing parentheses in front of the `||` to make the order to operations more explicit.